internal/dispatch/graph/check_test.go

```diff
@@ -27,7 +27,10 @@ import (
 	"github.com/authzed/spicedb/pkg/tuple"
 )
 
-var ONR = tuple.ONR
+var (
+	ONR = tuple.ONR
+	RR  = tuple.RR
```

Comment:
Why was this necessary?

Reply:
Ah, I take it this was just the place it made sense to move it within the package?

Reply:
It was in one of the deleted files but we still need it here, so I moved it